system/system_monitor/src/net_monitor/net_monitor.cpp

```diff
@@ -327,6 +338,62 @@ void NetMonitor::check_reassembles_failed(diagnostic_updater::DiagnosticStatusWr
   SystemMonitorUtility::stopMeasurement(t_start, status);
 }
 
+void NetMonitor::check_udp_buf_errors(diagnostic_updater::DiagnosticStatusWrapper & status)
+{
+  // Remember start time to measure elapsed time
+  const auto t_start = SystemMonitorUtility::startMeasurement();
+
+  int whole_level = DiagStatus::OK;
+  std::string error_message;
+
+  uint64_t total_udp_rcvbuf_errors = 0;
+  if (get_value_from_net_snmp(udp_rcvbuf_errors_index_, total_udp_rcvbuf_errors)) {
+    update_delta_queue(
+      udp_rcvbuf_errors_queue_, last_udp_rcvbuf_errors_, total_udp_rcvbuf_errors,
+      udp_buf_errors_check_duration_);
+    uint64_t unit_udp_rcvbuf_errors =
+      std::accumulate(udp_rcvbuf_errors_queue_.begin(), udp_rcvbuf_errors_queue_.end(), 0);
+    status.add(fmt::format("total UDP rcv buf errors"), total_udp_rcvbuf_errors);
+    status.add(fmt::format("UDP rcv buf errors per unit time"), unit_udp_rcvbuf_errors);
```

Comment:
I don't think fmt::format is necessary here.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/9538/commits/bfbbaac79edc8d02498b363e07098a4e0725ee12